src/Spotify.js

```diff
@@ -0,0 +1,37 @@
+import { URL } from 'url';
+
+const includesSomeOfArray = (string, array) =>
+  array.some(item => string.startsWith(item));
```

Comment:
Since the function's called `includesSomeOfArray`, we should use `includes` instead.
This so we can explicitly ignore embed-links (for now). (see below)

```suggestion
  array.some(item => string.includes(item));
```

Reply:
Hey sorry, I forgot to rename the function. But if we use includes than `/embed/podcastType` is also considered for transform URL so I intentionally used `startsWith`. Few tests will fail if we use `includes`

Reply:
Okay just checked the below suggestion. Sounds good 👍